red-hat-storage/ocs-ci
# Automation of intransit encryption sanity test.

Added a tests case to sanity check of intransit encryption feature.

Test Steps:
        1. Verify intransit Encryption is Enable on setup.
        2. Disable Intransit Encryption from setup.
        3. Verify config is removed from the cluster.
        4. re-enable intransit encryption on setup.
        5. Again verify intransit encryption config is added.
